cvat-ui/src/components/quality-control/task-quality/allocation-table.tsx

```diff
@@ -52,7 +53,16 @@ function AllocationTable(props: Readonly<Props>): JSX.Element {
     const data = validationLayout.validationFrames.map((frame: number, index: number) => ({
         key: frame,
         frame,
-        name: gtJobMeta.frames[index]?.name ?? gtJobMeta.frames[0].name,
+        name: gtJobMeta.frames[
+            (validationLayout.mode === ValidationMode.GT) ? frame : index
+            // The generic formula was here:
+            // https://github.com/cvat-ai/cvat/blob/45e1b4be5f40667d254cb869395f4cfa7dafc3ad/cvat-ui/ ...
+            // ... /src/components/quality-control/task-quality/allocation-table.tsx#L43
+            //
```

Comment:
Is this link necessary here? It seems like the comment below provides sufficient context.

Reply:
Ok, let's simplify it.